doc/classes/ProjectSettings.xml

```diff
@@ -1315,6 +1315,14 @@
 		<member name="input/ui_text_select_word_under_caret.macos" type="Dictionary" setter="" getter="">
 			macOS specific override for the shortcut to select the word currently under the caret.
 		</member>
+		<member name="input/ui_text_skip_selection_for_next_occurrence" type="Dictionary" setter="" getter="">
+			If no selection is currently active with the last caret in text fields, searches for the next occurrence of the the word currently under the caret and moves the caret to the next occurrence.
+			If a selection is currently active with the last caret in text fields, searches for the next occurrence of the selection, adds a caret, selects the next occurrence then deselects the previous selection and its associated caret.
+			Without selection, the action can be performed sequentially for other occurrences of the word under the last caret.
+			With selection, the action can be performed sequentially for other occurrences of the selection of the last caret.
+			The viewport is adjusted to the latest newly added caret.
```

Comment:
These lines should be joined together more, look at the documentation in the editor and you'll see, each new line forms a new paragraph

Reply:
I mimicked the documentation of [input/ui_text_add_selection_for_next_occurrence](https://github.com/godotengine/godot/blob/1c8ae43efefa679ed01ddefb38ccf2c1e52abb22/doc/classes/ProjectSettings.xml#L1135-L1140) to be as consistent as possible, but I might have missed the readability point.

Do a rework like this if preferable according to you ?
```
If no selection is currently active with the last caret in text fields, searches for the next occurrence of the the word currently under the caret and moves the caret to the next occurrence. The action can be performed sequentially for other occurrences of the word under the last caret.
If a selection is currently active with the last caret in text fields, searches for the next occurrence of the selection, adds a caret, selects the next occurrence then deselects the previous selection and its associated caret. The action can be performed sequentially for other occurrences of the selection of the last caret.
The viewport is adjusted to the latest newly added caret.
```

Reply:
Didn't see that one, then it's fine for this PR and we can work on both later 🙂